FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/DataProcessor/ParquetDataProcessor.cs

```diff
@@ -103,12 +99,22 @@ private ParquetConverter ParquetConverter
 
             string inputContent = string.Join(
                 Environment.NewLine,
-                processedData.Values.Select(jsonObject => jsonObject.ToString(Formatting.None))
-                         .Where(result => CheckBlockSize(processParameters.SchemaType, result)));
+                processedData.Values.Select(jsonObject => {
+                    var str = jsonObject.ToString(Formatting.None);
+
+                    // add a log for large resource
+                    if (str.Length > LargeResourceSize)
+                    {
+                        _diagnosticLogger.LogInformation($"The resource size {str.Length} of resource id {FhirBundleParser.ExtractResourceId(jsonObject)} is larger than {LargeResourceSize} Bytes.");
+                        _logger.LogInformation($"The resource size {str.Length} of resource id {FhirBundleParser.ExtractResourceId(jsonObject)} is larger than {LargeResourceSize} Bytes.");
```

Comment:
Feel little confused here because each **jsonObject** is a domain FHIR resource rather than a bundle, maybe add a **FhirResourceParser** as a base class for **FhirBundleParser**, and add **ExtractResourceId** it?

Reply:
as resource id is not logged now. remove the function extractResourceId.